### PR TITLE
chore: add repository and author meta

### PR DESCRIPTION
**Pull request by diervo**
## Details
This will allow all crawlers to link properly to the repo.

### CODE REVIEW

File: packages/@lwc/errors/package.json [hunk 1]
```diff
@@ -2,6 +2,11 @@
     "name": "@lwc/errors",
     "version": "1.5.2",
     "description": "LWC Error Utilities",
+    "author": "LWC Team",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/salesforce/lwc.git"
+    },
```
Comment (ekashida) [inline packages/@lwc/errors/package.json:9]: This is a public-private package so I don't think we want to make it easier for crawlers to find it.

File: packages/@lwc/features/package.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
     "name": "@lwc/features",
     "version": "1.5.2",
     "description": "LWC Features Flags",
+    "author": "LWC Team",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/salesforce/lwc.git"
+    },
```
Comment (ekashida) [inline packages/@lwc/features/package.json:9]: This is a public-private package so I don't think we want to make it easier for crawlers to find it.

File: packages/@lwc/shared/package.json [hunk 3]
```diff
@@ -1,6 +1,11 @@
 {
     "name": "@lwc/shared",
     "version": "1.5.2",
+    "author": "LWC Team",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/salesforce/lwc.git"
+    },
```
Comment (ekashida) [inline packages/@lwc/shared/package.json:8]: This is a public-private package so I don't think we want to make it easier for crawlers to find it.

File: package.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "lwc",
+  "author": "LWC Team",
```
Comment (caridy) [inline package.json:3]: should this be a proper github team?

Comment (diervo) [reply package.json:3]: not sure if there is any magical auto linking... Dont think is important

**Comment by diervo** [comment]
I will be closing this PR and creating a new one since will be faster than resolving the conflicts.

**Comment by ekashida** [comment]
#1903

**Pull request closed without merging**